packages/hash/frontend/src/pages/[account-slug]/entities/[entity-uuid].page/entity-editor/properties-section/property-table/use-row-data/generate-property-rows-from-entity/generate-property-row-recursively.ts

```diff
@@ -35,14 +35,21 @@ import { getExpectedTypesOfPropertyType } from "./get-expected-types-of-property
  *
  * @returns property row (and nested rows as `children` if it's a nested property)
  */
-export const generatePropertyRowRecursively = (
-  properties: unknown,
-  propertyTypeBaseUri: string,
-  propertyKeyChain: string[],
-  entitySubgraph: Subgraph<SubgraphRootTypes["entity"]>,
-  requiredPropertyTypes: string[],
+export const generatePropertyRowRecursively = ({
+  propertyTypeBaseUri,
+  propertyKeyChain,
+  entity,
+  entitySubgraph,
+  requiredPropertyTypes,
   depth = 0,
-): PropertyRow => {
+}: {
+  propertyTypeBaseUri: string;
```

Comment:
Can we try and use the `BaseUri` type (it's just an alias for string so it is the same, but it's more descriptive)